src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is working");
```

Comment:
The message in the doWork method should be more informative. According to the checklist, it should indicate the type of machine that is working. Consider specifying that it is an Excavator that is working.